runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerConfig.java

```diff
@@ -77,12 +95,12 @@ default int pauseSkillAfter()
 
 	@ConfigItem(
 		position = 4,
-		keyName = "onScreenXpLeft",
-		name = "Display XP Left on-screen",
-		description = "Display remaining experience instead of experience gained on on-screen trackers"
+		keyName = "onScreenDisplayMode",
+		name = "On-screen tracker display mode",
+		description = "Configures how first line in on screen XP overlay is displayed"
```

Comment:
Maybe ``Configures the information displayed in the first line of on-screen XP overlays`` is better 😃